azure-pipelines.yml

```diff
@@ -1,7 +1,8 @@
 variables: {
   WORKER_VERSION: '1.1.1',
   NODE_LOWER_LTS: '8.x',
-  NODE_HIGHER_LTS: '10.x'
+  NODE_HIGHER_LTS: '10.x',
+  NODE_PREVIEW_LTS: '12.x'
```

Comment:
Node 12 is "Active LTS", not preview. I suppose you're saying this is preview for us though, right? That might be fine